spyder/widgets/variableexplorer/collectionseditor.py

```diff
@@ -511,6 +514,17 @@ def create_dialog(self, editor, data):
                      lambda eid=id(editor): self.editor_rejected(eid))
         editor.show()
         
+    @Slot(str, object)
+    def change_option(self, option_name, new_value):
+        """
+        Change configuration option.
+
+        This function is called when a `sig_option_changed` signal is received.
+        At the moment, this signal can only come from a DataFrameEditor.
+        """
+        assert option_name == 'dataframe_format'
+        self.parent().set_dataframe_format(new_value)

```

Comment:
Could we move this method to `dataframeeditor`? It has nothing to do with `collectionseditor` :-)